packages/pyright-internal/src/languageService/analyzerServiceExecutor.ts

```diff
@@ -148,6 +148,11 @@ function getEffectiveCommandLineOptions(
         commandLineOptions.configSettings.verboseOutput = true;
     }
 
+    if (serverSettings.fileEnumerationTimeoutInSec) {
```

Comment:
currently if you set it to 0, it sets the value to 10 (the default) instead
```suggestion
    if (serverSettings.fileEnumerationTimeoutInSec !== undefined) {
```